packages/react/src/components/Accordion/Accordion.js

```diff
@@ -56,6 +58,11 @@ Accordion.propTypes = {
    */
   disabled: PropTypes.bool,
 
+  /**
+   * Specify whether an individual AccordionItem should be flush, default is false
+   */
+  isFlush: PropTypes.bool,
```

Comment:
Is it possible to make only one AccordionItem flush or will `isFlush` make all items flushed?

Reply:
@abbeyhrt I guess that's a design question. I assumed it would be for the accordion as a whole like we do for the prop `align`. @laurenmrice is the intent that the prop would be for a specific `AccordionItem` or the whole `Accordion`?

Reply:
We would want the whole accordion to be `isFlush`, not for each individual item.

Reply:
That makes sense! Could we update the proptype comment here to something like "Specify whether Accordion should be flush"?

Reply:
ah yes, totally missed updating the `AccordionItem` comment, I had just copied it from the disabled proptype. Great catch @abbeyhrt!